_DeploymentAndDistroPackaging/deb/debian/control

```diff
@@ -0,0 +1,86 @@
+Source: ciao
+Section: devel
+Priority: extra
+Maintainer: Erick Cardona <erick.cardona.ruiz@intel.com>
+Uploaders: Erick Cardona <erick.cardona.ruiz@intel.com>
+Build-Depends: debhelper (>= 9),
+               dh-golang,
+               golang-go (>= 1.6.2),
+               golang-github-google-gofuzz-dev,
+               golang-github-stretchr-testify-dev,
+               git,
+               ca-certificates,
+               golang-glog-dev,
+               golang-gopkg-yaml.v2-dev,
+               golang-github-mitchellh-mapstructure-dev,
+               golang-github-rackspace-gophercloud-dev,
+               golang-github-gorilla-mux-dev,
+               golang-github-mattn-go-sqlite3-dev,
+               golang-github-boltdb-bolt-dev,
+               golang-github-coreos-go-iptables-dev,
+               golang-github-docker-engine-api-dev,
+               golang-github-docker-libnetwork-dev,
+               golang-github-vishvananda-netlink-dev,
+               golang-golang-x-net-dev,
+               golang-github-docker-docker-dev
+
+Standards-Version: 3.9.7
+Homepage: https://github.com/01org/ciao
+XS-Go-Import-Path: github.com/01org/ciao
+Package: ciao
+Architecture: amd64
+Built-Using: ${misc:Built-Using}
+Depends: ${shlibs:Depends},
+         ${misc:Depends}
+Description: Ciao - Cloud Integrated Advanced Orchestrator
```

Comment:
Is it normal in debian to have this verbose of a description?  With this much detail, it will be more difficult to maintain and keep in sync with the project.  At least Go Report Card should not be the first line of the description.